meteor/server/systemStatus/systemStatus.ts

```diff
@@ -295,12 +296,24 @@ export async function getSystemStatus(cred0: Credentials, studioId?: StudioId):
 		statusObj.components.push(so)
 	}
 
+	const versions: { [name: string]: string } = await RelevantSystemVersions
```

Comment:
should `versions` be cloned before being mutated?
It looks like `await RelevantSystemVersions` will return the same object every time, so your array below will be mutating this cached object